### PR TITLE
Ignore files list

### PR DESCRIPTION
**Pull request by uwedeportivo**
input files matching glob pattern specified by this flag are ignored

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -163,7 +165,7 @@ func loadResource(rootDir string, filename string) (*Resource, error) {
 
 	labels, ok := metadata["labels"].(map[string]interface{})
 	if !ok {
-		return nil, fmt.Errorf("resource %s is missing labels section", filename)
+		labels = make(map[string]interface{})
```
Comment (ggilmore) [inline main.go:168]: How is this change related?

File: main.go [hunk 2]
```diff
@@ -256,6 +258,15 @@ func loadResourceSet(inputs []string) (*ResourceSet, error) {
 			}
 
 			if filepath.Ext(path) == ".yaml" || filepath.Ext(path) == ".yml" {
+				for _, ignorePattern := range ignoreFiles {
+					ignore, err := filepath.Match(ignorePattern, filepath.Base(path))
```
Comment (ggilmore) [inline main.go:262]: Is this actually correct? What happens if I want to ignore everything under a certain folder? This only seems to consider the last element of the path.

**Review by ggilmore — CHANGES_REQUESTED** [review]

File: main.go (hunk 2 above)
Comment (uwedeportivo) [reply main.go:262]: that is correct, i was lazy :-)

Comment (uwedeportivo) [reply main.go:262]: i did just enough to be able to run it over the whole deploy-sourcegraph dir

Comment (uwedeportivo) [reply main.go:262]: i'll fix it

File: main.go (hunk 1 above)
Comment (uwedeportivo) [reply main.go:168]: it's not, i should transfer it into the component_labeling PR. i stacked the PRs and found the bug while working on this part

Comment (uwedeportivo) [reply main.go]: done

File: main.go (hunk 2 above)
Comment (uwedeportivo) [reply main.go]: done
